docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,501 @@
+   "execution_count": null,
+   "id": "3cceeefb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_examples_to_show = 5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4caa635d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_issue_idx = find_label_issues(labels, predictions, return_indices_ranked_by_score=True)\n",
+    "label_issue_idx[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "638a9dc4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "is_label_issue = find_label_issues(labels, predictions)\n",
+    "is_label_issue[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b501dc9",
+   "metadata": {},
+   "source": [
+    "### Get label quality scores\n",
+    "The above example shows how to identify which examples have label quality issues. We can also compute scores for each example in the dataset which estimate our confidence that this example has been correctly labeled using `get_label_quality_scores()`. These scores range between 0 and 1 with smaller values indicating examples whose annotation seems more likely to be imprefect.\n",
+    "\n",
+    "This method is useful for prioritizing which images to review if you have too little time to review all of the label issues by reviewing the images with the lowest scores first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a9b4c590",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scores = get_label_quality_scores(labels, predictions)\n",
+    "scores[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "349521e0",
+   "metadata": {},
+   "source": [
+    "We can also use `issues_from_scores()` which returns an array of issue indices sorted from most to least severe who's label quality scores fall below the threshold if one is provided."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ffd9ebcc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "issue_idx = issues_from_scores(scores, threshold=0.5)\n",
+    "issue_idx[:num_examples_to_show], scores[issue_idx][:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a3b8aa0",
+   "metadata": {},
+   "source": [
+    "## 4. Use ObjectLab to visualize label issues\n",
+    "Finally, we can use cleanlab functionality to visualize the potential issue errors. Lets take a closer look at an example with a low score using cleanlab's `visualize()` function. To make analysis easier we can also input a `class_names` dictionary into the function to show up as a legend and turn off `overlay` to see the given and predicted labels side by side."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4dd46d67",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "example_issue_idx = issue_idx[10]\n",
```

Comment:
@ulya-tkch need to add test in the final hidden cell that:

this specific image was the 10th example listed in `issue_idx`